File: EX5 revisao/relacionais/valor_numero.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner gi = new Scanner(System.in);

        System.out.print("Digite um número: ");
        int num = gi.nextInt();

        if (num > 0)
            System.out.println("O número " + num +" é positivo.");
        else if (num < 0)
            System.out.println("O número " + num +" é negativo.");
        else
            System.out.println("O número é zero.");
    }
}

File: EX5 revisao/relacionais/valor_numero.cpp
#include <iostream>
using namespace std;

int main() {
    float n;
    cout << "Digite um número: \n";
    cin >> n;
    
    if(n < 0)
    cout << "O numero "<< n<<" é negativo\n";
    else if(n > 0)
    cout << "O numero "<< n <<" é positivo\n";
    else
    cout << " O numero é zero\n";
}